drivers/UARTSerial.h

```diff
@@ -157,6 +157,12 @@ class UARTSerial : private SerialBase, public FileHandle {
      */
     void set_data_carrier_detect(PinName dcd_pin, bool active_high = false);
 
+    /** Set the baud rate
+     *
+     *  @param baud   The baud rate
+     */
+    void set_baud(int baud);
```

Comment:
@kjbracey-arm: Kevin, are you OK with the convention of simply calling in the `SerialBase` method rather than having the intermediate that adds 'set_'?